### PR TITLE
On registration, add a "harvest by CORE" logo

### PR DESCRIPTION
**Pull request by samuelpearce**
When a user successfully registers for a repository, suggest to the user
to add a "Harvested by CORE" logo on their website

### CODE REVIEW

File: templates/data-providers/utils/generate-form-message.jsx [hunk 1]
```diff
@@ -35,7 +36,11 @@ const generateFormMessage = ({ created, duplicated, error }) => {
           >
             {created.email}
           </Link>
-          .
+          . Join the community and add a{' '}
+          <Link href={HARVESTED_BY_CORE} title="Harvested by CORE Logo">
+            harvested by CORE
+          </Link>
+          badge on your website.
```
Comment (mcancellieri) [inline templates/data-providers/utils/generate-form-message.jsx:43]:
```suggestion
           badge on your website.
```
Missing a space

**Review by mcancellieri — APPROVED** [review]
Just remember to add the space and ready to go :)

File: templates/data-providers/utils/generate-form-message.jsx (hunk 1 above)
Comment (valerapro) [reply templates/data-providers/utils/generate-form-message.jsx:43]: Nope, it wrong according to `eslint`. Sam fix works

**Review by valerapro — APPROVED** [review]

File: templates/data-providers/utils/generate-form-message.jsx (hunk 1 above)
Comment (mcancellieri) [reply templates/data-providers/utils/generate-form-message.jsx:43]: ok, maybe needs a `&nbsp;`?

Even if the linter disagree it is missing a space :)

Comment (samuelpearce) [reply templates/data-providers/utils/generate-form-message.jsx:43]: @mcancellieri Thanks for spotting. You might be looking at a old commit though.
It was fixed in [35a824c](https://github.com/oacore/search/pull/187/commits/35a824c9e8bbcbf4e25c94b92e68320bd2c15013) which was included in @valerapro merge

**Pull request merged**